cmd/limactl/guest_install.go

```diff
@@ -0,0 +1,116 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/lima-vm/lima/pkg/cacheutil"
+	"github.com/lima-vm/lima/pkg/store"
+	"github.com/lima-vm/lima/pkg/store/filenames"
+	"github.com/lima-vm/lima/pkg/usrlocalsharelima"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+func newGuestInstallCommand() *cobra.Command {
+	guestInstallCommand := &cobra.Command{
+		Use:               "guest-install INSTANCE",
+		Short:             "Install guest components",
+		Args:              WrapArgsError(cobra.MaximumNArgs(1)),
+		RunE:              guestInstallAction,
+		ValidArgsFunction: cobra.NoFileCompletions,
+		Hidden:            true,
```

Comment:
Maybe this should be grouped under something like `limactl alpha`, `limactl devtools`, `limactl debugtools`

Reply:
It is not a development or debugging tool, but we could put it in some group if necessary